packages/server-core/src/route/route/route.test.ts

```diff
@@ -0,0 +1,104 @@
+import appRootPath from 'app-root-path'
+import assert from 'assert'
+import fs from 'fs'
+import path from 'path'
+import { v4 as uuid } from 'uuid'
+
+import { destroyEngine } from '@etherealengine/engine/src/ecs/classes/Engine'
+
+import { Application } from '../../../declarations'
+import { createFeathersExpressApp } from '../../createApp'
+import { deleteFolderRecursive } from '../../util/fsHelperFunctions'
+
+const params = { isInternal: true } as any
+
+const cleanup = async (app: Application, projectName: string) => {
+  const projectDir = path.resolve(appRootPath.path, `packages/projects/projects/${projectName}/`)
+  deleteFolderRecursive(projectDir)
+  try {
+    await app.service('project').Model.destroy({ where: { name: projectName } })
+  } catch (e) {}
+}
+
+const updateXREngineConfigForTest = (newProject: string) => {
+  const testXREngineConfig = `
+  import type { ProjectConfigInterface } from '@etherealengine/projects/ProjectConfigInterface'
+
+  const config: ProjectConfigInterface = {
+    onEvent: undefined,
```

Comment:
we can make this leaner by removing all properties except `routes`

Reply:
yes, not all properties are required by the `xrengine.config`.
added this change!